### PR TITLE
feat: add chaos mesh api description and table schema

### PR DESCRIPTION
**Pull request by kerenfi**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                      |
| Breaking change |✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |

Related issues: #634 

Further  information:

### CODE REVIEW

File: src/chaos-experiments/models/chaosExperimentsManager.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const contextId = httpContext.get(CONTEXT_ID);
+
+    const chaosExperimentWithTheSameName = await databaseConnector.getChaosExperimentByName(chaosExperiment.name, contextId);
+    if (chaosExperimentWithTheSameName) {
+        throw generateError(400, ERROR_MESSAGES.CHAOS_EXPERIMENT_NAME_ALREADY_EXIST);
+    }
+    const experimentId = uuid.v4();
+    try {
+        await databaseConnector.insertChaosExperiment(experimentId, chaosExperiment, contextId);
+        chaosExperiment.id = experimentId;
```
Comment (GuyAb) [inline src/chaos-experiments/models/chaosExperimentsManager.js:21]: why do you need it here? I would set the id before sending to db connector, this way you have to set it twice

File: tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+                should(getChaosExperimentsResponse.statusCode).equal(200);
+                const chaosExperiments = getChaosExperimentsResponse.body;
+                should(chaosExperiments.length).equal(3);
+
+                const contextAResponse = getChaosExperimentsResponse.body.find(o => o.id === chaosExperimentResponse.body.id);
+                should(contextAResponse).not.be.undefined();
+            });
+
+            it('get chaos experiments with wrong context_id should no processors', async function () {
```
Comment (GuyAb) [inline tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js:46]: not processors

File: tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+                const headers = {
+                    'Content-Type': 'application/json',
+                    'x-context-id': uuid.v4()
```
Comment (GuyAb) [inline tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js:49]: We must have context-id to use the API? Is is wanted?

File: tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            beforeEach(async function () {
+                const chaosExperiment = generateRawChaosExperiment(uuid.v4(), contextId);
+                const headersWithContext = Object.assign({}, validHeaders, { 'x-context-id': contextId });
+
+                chaosExperimentResponse = await chaosExperimentsRequestSender.createChaosExperiment(chaosExperiment, headersWithContext);
+                experimentId = chaosExperimentResponse.body.id;
+            });
+
+            afterEach(async function () {
+                await chaosExperimentsRequestSender.deleteChaosExperiment(experimentId, { 'Content-Type': 'application/json' });
```
Comment (GuyAb) [inline tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js:85]: why delete in after each if you already deleted in it section?

File: tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js (hunk 3 above)
Comment (kerenfi) [reply tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js:49]: no...it's not required the predator allows it as like overall context

File: src/chaos-experiments/models/chaosExperimentsManager.js (hunk 1 above)
Comment (kerenfi) [reply src/chaos-experiments/models/chaosExperimentsManager.js:21]: I think it's more consistent to put it here - just looked on other resources. don't have a strong opinion here

File: tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js (hunk 2 above)
Comment (kerenfi) [reply tests/integration-tests/chaos-experiments/chaos-experiments-context-test.js:46]: done

File: docs/openapi3.yaml [hunk 5]
```diff
@@ -2037,6 +2037,49 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error_response'
+    put:
+      operationId: update-chaos-experiment
+      tags:
+        - Chaos Experiments
+      summary: Delete chaos experiment
+      description: Delete a specific chaos experiment
```
Comment (GuyAb) [inline docs/openapi3.yaml:2045]: not delete

Comment (kerenfi) [reply docs/openapi3.yaml]: done

**Pull request merged**